x-pack/plugins/osquery/server/routes/live_query/create_live_query_route.ts

```diff
@@ -95,7 +97,11 @@ export const createLiveQueryRoute = (router: IRouter, osqueryContext: OsqueryApp
         const { response: osqueryAction } = await createActionHandler(
           osqueryContext,
           request.body,
-          { soClient, metadata: { currentUser } }
+          {
+            soClient,
+            metadata: { currentUser },
+            ecsData: { ecs: { version: '8.6.0' }, ...alertData } as Ecs,
```

Comment:
I think that we should pass `ecsData` only if alertData exists.  Otherwise we will trigger https://github.com/elastic/kibana/blob/e38350f7f930ec8adb924bd92a822bcf70340d52/x-pack/plugins/osquery/server/handlers/action/create_action_handler.ts#L106 this one with no data.